code/game/machinery/doors/airlock_types.dm

```diff
@@ -298,6 +298,17 @@
 	opacity = 0
 	glass = TRUE
 
+/obj/machinery/door/airlock/bronze
+	name = "bronze airlock"
+	icon = 'icons/obj/doors/airlocks/clockwork/pinion_airlock.dmi'
+	overlays_file = 'icons/obj/doors/airlocks/clockwork/overlays.dmi'
+	assemblytype = /obj/structure/door_assembly/door_assembly_bronze
+	anim_parts = "left=-13,0;right=13,0"
```

Comment:
Give these no armour and weak health.

Reply:
Is there any reason why they should be weaker than normal airlock types? You still need electronics to build them

Reply:
Because they are made of brass (And would be stronger than bronze airlocks otherwise due to deconstruction)